src/osd/osd_types.h

```diff
@@ -2315,13 +2315,15 @@ struct pg_query_t {
     LOG = 1,
     MISSING = 4,
     FULLLOG = 5,
+    ALL = 6,
```

Comment:
They use different messages, but the MOSDPGLog message returned from FULLLOG request includes the info.  I'd prefer you not change the messages if possible since it would make handling and testing the mixed cluster case much harder.